core/src/optimistic_confirmation_verifier.rs

```diff
@@ -61,15 +61,6 @@ impl OptimisticConfirmationVerifier {
             return;
         }
 
-        datapoint_info!(
```

Comment:
I've never looked at this metric, but defering to @steviez's opinion since he's been more involved in replay than I have

Reply:
This one is kind of interesting, but I don't check it very often and I don't think we need every node reporting it. This information can be determined by other means (RPC, ledger-tool) as well.

That being said, the value is currently read in this dashboard:
https://github.com/anza-xyz/agave/blob/a72f981370c3f566fc1becf024f3178da041547a/metrics/scripts/grafana-provisioning/dashboards/cluster-monitor.json#L10705

Maybe we can remove it from the dashboard as part of this PR too ?

Reply:
I think we will have a new dashboard after the metric audit, but it's good to remove it from the file to avoid any misleading info! [0bef5e8](https://github.com/anza-xyz/agave/pull/2717/commits/0bef5e830759bd7001180cd525c9d276a2a4c34e)